ipywidgets/static/widgets/js/layout.js

```diff
@@ -36,18 +36,15 @@ define([
                 'align_items',
                 'align_self',
                 'bottom',
+                'border',
```

Comment:
I included it because it is part of the [css box model](http://www.w3.org/TR/CSS21/images/boxdim.png).